src/core_modules/capture-core/components/D2Form/FormBuilder/FormBuilder.component.js

```diff
@@ -500,6 +531,31 @@ export class FormBuilder extends React.Component<Props> {
         }, []);
     }
 
+    renderPlugin = (
+        field: Object,
```

Comment:
I believe the type can be improved to `field: FieldConfig` if `field.props` is checked for null or undefined [below](https://github.com/dhis2/capture-app/pull/3222/files#diff-34437e7d08816ce5e6cc3c54623ac10f394f1c42f131e84664f15a12902b3e7dR545). WDYT? Thanks!